src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/VirtualSecureModeEnclaveProvider.cs

```diff
@@ -86,7 +93,201 @@ protected override byte[] MakeRequest(string url)
             throw SQL.AttestationFailed(string.Format(Strings.GetAttestationSigningCertificateRequestFailedFormat, url, exception.Message), exception);
         }
 
+        private static int DecodeUtf8JsonBytes(Stream input, out byte[] bytes)
```

Comment:
I didn't expect the CI to succeed. I wanted to see what the json looked like since the spec is extremely light on important details like encoding and formatting.

If json is more broadly required then we should take the dependency. For people not using json support (most of them) it would still be a good idea to be able to get rid of the dependency entirely.

This isn't just json though it's the whole DataContractJsonDeserializer and all the attributes reflection etc that it roots. It'd arguably be better to use S.T.Json. I understand why it was written this way, the author didn't have enough detail to do it in a more careful way. I suspect that it mirrors the implementation types and all.